maven-core/src/main/java/org/apache/maven/toolchain/DefaultToolchainManagerPrivate.java

```diff
@@ -39,6 +43,19 @@
     extends DefaultToolchainManager
     implements ToolchainManagerPrivate
 {
+    @Inject
+    public DefaultToolchainManagerPrivate()
+    {
+        super();
+    }
+
+    /**
+     * Ctor for UT.
+     */
+    public DefaultToolchainManagerPrivate( Logger logger )
```

Comment:
If it is for a unittest, can you remove public?

Reply:
fixed